share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -1,101 +1,50 @@
 
+Packages are installed according to their formulae, which live in `/usr/local/Library/Formula`. Check one out a simple one e.g. `brew edit etl` (or [etl](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/etl.rb)) or a more advanced one e.g. `brew edit git` or [Git](https://github.com/Homebrew/homebrew/tree/master/Library/Formula/git.rb).
 
 # Basic Instructions
 
 Make sure you run `brew update` before you start. This turns your Homebrew installation into a Git repository.
 
-Before contributing, make sure your package:
+Before submitting a new formula make sure your package:
 
 *   meets all our [Acceptable Formulae](Acceptable-Formulae.md) requirements
 *   isn't already in Homebrew (check `brew search $FORMULA`)
-*   isn't in another [Homebrew tap](https://github.com/Homebrew)
+*   isn't in another official [Homebrew tap](https://github.com/Homebrew)
 *   isn't already waiting to be merged (check the [issue tracker](https://github.com/Homebrew/homebrew/issues))
-*   is still supported by upstream
+*   is still supported by upstream (i.e. doesn't require extensive patching)
 *   has a stable, tagged version (i.e. not just a GitHub repository with no versions). See [Interesting-Taps-&-Branches](Interesting-Taps-&-Branches.md) for where pre-release versions belong.
 *   passes all `brew audit --strict --online $FORMULA` tests.
 
-Make sure you search thoroughly (all aliases!). We don’t want you to waste your time.
-
-Be sure to look over the [contributing guidelines](https://github.com/Homebrew/homebrew/blob/master/CONTRIBUTING.md) as well.
-
-
-## Will we merge your formula?
-
-Probably. But we have rules to keep the quality and goals of Homebrew intact: Please read [Acceptable Formulae](Acceptable-Formulae.md).
-
-## Some Quick Examples Before You Get Started
-
-Formulae aren’t that complicated. [etl](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/etl.rb) is as simple as it gets.
-
-And then [Git](https://github.com/Homebrew/homebrew/tree/master/Library/Formula/git.rb) and [flac](https://github.com/Homebrew/homebrew/tree/master/Library/Formula/flac.rb) show more advanced functionality.
+Before submitting a new formula make sure you read over our [contribution guidelines](https://github.com/Homebrew/homebrew/blob/master/CONTRIBUTING.md).
 
 ## Grab the URL
 
-All you need to make a formula is a URL to the tarball.
-
-    brew create https://example.com/foo-0.1.tar.gz
+Run `brew create` with a URL to the source tarball:
 
-This creates:
-
-`$HOMEBREW_REPOSITORY/Library/Formula/foo.rb`
+```shell
+brew create https://example.com/foo-0.1.tar.gz
+```
 
-And opens it in your `$EDITOR`. It'll look like:
+This creates `/usr/local/Library/Formula/foo.rb` and opens it in your `$EDITOR`. It'll look something like:
 
 ```ruby
 class Foo < Formula
```

Comment:
Tiny nit: The example ruby block here is wrong. Should be desc => homepage => url => sha256, not url => homepage => sha256.